types/govuk-frontend/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/govuk-frontend",
-    "version": "5.4.9999",
+    "version": "5.6.9999",
```

Comment:
I've skipped v5.5.0 intentionally as the exported types didn't change until v5.6.0